File: CS/WebDesigner_DataLoading/SalesPersonData.cs
using System;
using System.Collections.Generic;
using System.Threading;

namespace WebDesigner_DataLoading {
    public class SalesPersonData {
        public string SalesPerson { get; set; }
        public int Quantity { get; set; }

        public static List<SalesPersonData> CreateData() {
            List<SalesPersonData> data = new List<SalesPersonData>();
            string[] salesPersons = { "Andrew Fuller", "Michael Suyama", 
                                    "Robert King", "Nancy Davolio",
                                    "Margaret Peacock", "Laura Callahan", 
                                    "Steven Buchanan", "Janet Leverling" };

            for (int i = 0; i < 100; i++) {
                SalesPersonData record = new SalesPersonData();
                int seed = (int)DateTime.Now.Ticks & 0x0000FFFF;
                record.SalesPerson = salesPersons[new Random(seed).Next(0, salesPersons.Length)];
                record.Quantity = new Random(seed).Next(0, 100);
                data.Add(record);
                Thread.Sleep(3);
            }
            return data;
        }
    }
}

File: VB/WebDesigner_DataLoading/SalesPersonData.vb
Imports System
Imports System.Collections.Generic
Imports System.Threading

Namespace WebDesigner_DataLoading
	Public Class SalesPersonData
		Public Property SalesPerson() As String
		Public Property Quantity() As Integer

		Public Shared Function CreateData() As List(Of SalesPersonData)
		
			Dim data As New List(Of SalesPersonData)()
			Dim salesPersons() As String = {"Andrew Fuller", "Michael Suyama",
                                            "Robert King", "Nancy Davolio",
                                            "Margaret Peacock", "Laura Callahan",
                                            "Steven Buchanan", "Janet Leverling"}
			Dim rnd = New Random()
			For i As Integer = 0 To 99
				Dim record As New SalesPersonData()
				record.SalesPerson = salesPersons(rnd.Next(0, salesPersons.Length))
				record.Quantity = rnd.Next(0, 100)
				data.Add(record)
			Next i
			Return data
		End Function
	End Class
End Namespace